### PR TITLE
Filter broadcasts without crids from controller output

### PR DESCRIPTION
**Pull request by tfm**
These would never be uploaded to YouView anyway, and break
since later code relies on them being present

### CODE REVIEW

File: src/main/java/org/atlasapi/query/v2/ContentFeedController.java [hunk 1]
```diff
@@ -119,8 +123,10 @@ public void generateFeed(HttpServletRequest request, HttpServletResponse respons
             if (content.get() instanceof Item) {
                 Item item = (Item) content.get();
                 
+                Predicate<ItemBroadcastHierarchy> broadcastFilter = FilterFactory.broadcastFilter(Optional.<DateTime>absent());
```
Comment (oli-hall) [inline src/main/java/org/atlasapi/query/v2/ContentFeedController.java]: you might want to check what that does when the DateTime not present...

**Comment by oli-hall** [comment]
back to you, @tfm

File: src/main/java/org/atlasapi/query/v2/ContentFeedController.java (hunk 1 above)
Comment (tfm) [inline src/main/java/org/atlasapi/query/v2/ContentFeedController.java]: I did, I think it does the right thing. Are you saying you think it doesn't?

Comment (oli-hall) [inline src/main/java/org/atlasapi/query/v2/ContentFeedController.java]: If no DateTime, it returns ``Predicates.alwaysTrue()``. In that case, there's no point in having it, as it gives nothing extra. Afaik, the only point in running the extra filtering is to get the Pid filtering, which this won't do.

Comment (tfm) [inline src/main/java/org/atlasapi/query/v2/ContentFeedController.java]: Fixed the predicate :)

**Pull request merged**